nixos/modules/services/networking/minidlna.nix

```diff
@@ -108,6 +108,29 @@ in
           Many people prefer shorter announce intervals (e.g. 60 seconds)
           on their home networks, especially when DLNA clients are
           started on demand.
+
+          Instead of waiting on announces, one can open the port UDP 1900
+          to use SSDP discovery. Furthermore setting announce interval
+          to 90000 may be a solution if you are experiencing disconnects
+          with certain clients.
+        '';
+    };
+
+    services.minidlna.inotify = mkOption {
+      type = types.str;
```

Comment:
You can use enum for available values instead of str:
```nix
{
  type = types.enum [ "yes" "no" ];
}
```

Reply:
Good idea